tests/packages/link-library-in-subproject/subprojects/bar/examplelib.c

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2022 The meson-python developers
+//
+// SPDX-License-Identifier: MIT
+
+#include "bar_dll.h"
+
+BAR_DLL int sum(int a, int b) {
+	return a + b;
```

Comment:
It does not matter much, but we use 4 spaces indentation for `.c` and `.cpp` files in all other files, keeping consistency is always nice 🙂